Makefile

```
@@ -20,7 +20,7 @@ SHELL := /usr/bin/env bash
 
 # release images are available at:
 # console.cloud.google.com/gcr/images/cloud-foundation-cicd/global/cft/developer-tools?project=cloud-foundation-cicd
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.4.13
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.10.6
```

The linter has an issue in 1.9 and 1.10.x, so I'd recommend using most recent 1.8 or 1.11

```suggestion
DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.11.4
```